.gitlab/deps_build.yml

```diff
@@ -97,3 +97,23 @@ build_clang_arm64:
   tags: ["runner:docker-arm", "platform:arm64"]
   variables:
     ARCH: arm64
+
+.build_omnibus-nikos_common:
+  stage: deps_build
+  script:
+    - git clone https://github.com/DataDog/omnibus-nikos.git /tmp/omnibus-nikos
+    - cd /tmp/omnibus-nikos
+    - /bin/bash -l -c "bundle install"
+    - /bin/bash -l -c "bundle exec omnibus build nikos"
```

Comment:
How long does the nikos build take? If it takes awhile, we should make this a manual job, make the artifacts permanent, and remove the job dependency on `build_system-probe-x64`.

Reply:
If I remember correctly, it takes 20-30 minutes - so it's not super long but honestly there isn't a good reason to be running that job for every build anyways, so I'll make it manual.